website/site/content/docs/configuration-options.md

```diff
@@ -100,7 +100,9 @@ You may also specify a custom `extension` not included in the list above, as lon
 - `toml-frontmatter`: same as the `frontmatter` format above, except frontmatter will be both parsed and saved only as TOML, followed by unparsed body text
 - `json-frontmatter`: same as the `frontmatter` format above, except frontmatter will be both parsed and saved as JSON, followed by unparsed body text
 
-The explicit `yaml-frontmatter`, `toml-frontmatter`, and `json-frontmatter` formats above do not currently support custom delimiters. We use `---` for YAML, `+++` for TOML, and `{` `}` for JSON. If a file has frontmatter inside other delimiters it will be included as part of the body text.
+The explicit `yaml-frontmatter`, `toml-frontmatter`, and `json-frontmatter` formats support custom delimiters. We use `---` for YAML, `+++` for TOML, and `{` `}` for JSON by default, but you can configure a custom delimiter.
+You can configure a custom delimiter with `frontmatter_delimiter`, but it only applies if you have a frontmatter format specified.
+If a file has frontmatter inside other delimiters it will be included as part of the body text.
```

Comment:
Thanks for including this change in the docs!

Could you format this like the other config options? Something like:

> - `frontmatter_delimiter`:  *description here*

I think you could also add the default delimiters (`+++` etc) to the descriptions for the explicit frontmatter formats instead of putting them all together in the `frontmatter_delimiter` description.

Reply:
@verythorough I took your comments into consideration, I hope the changes reflect the direction you were looking for.